src/store/modules/app.ts

```diff
@@ -154,12 +155,29 @@ const actions = {
     try {
       const blockNumber = await getBlockNumber(getProvider(space.network));
       const result: any = {};
-      const [proposal, votes] = await Promise.all([
+      const [proposal, votes]: [any, any] = await Promise.all([
         ipfsGet(gateway, id),
         client.request(`${space.key}/proposal/${id}`)
       ]);
+      const profile = await Box.getProfile(proposal.address)
```

Comment:
It will be faster to resolve if you load the author profile within the Box.profileGraphQL request, we would have a single request instead of 2.